docs/api/covidcast-signals/nssp.md

```diff
@@ -0,0 +1,100 @@
+
+
+## Limitations
+
+There is substantial missingness at the county level. This tends to impact more rural and lower population locations. See the [missingness section below](#missingness) for more information.
+
+NSSP notes that not every patient entering an ED is tested for the conditions of interest, so the data may undercount total cases and as a result underreport percent visits.
+
+Our [geographic weighting approach](#geographic-weighting) assumes that the number of ED visits is proportional to the overall population of a county. However, in reality, there are various factors that could affect the accuracy of this assumption.
+
+For example, we might expect denser, more urban counties to have 1) more and larger EDs and 2) easier access to EDs. The first factor may mean that residents of rural counties are more likely to go to EDs in urban counties. The second factor may increase the total number of ED visits that someone living in an urban county will make, that is, the average urban resident may make more ED visits than the average rural resident.
+
+As a result, total ED visits per capita in rural counties may be lower than total ED visits per capita in urban counties. Since our weighting approach uses population as the weighting factor, rural counties would tend to be overrepresented in estimated values.
+
+Some low population counties occasionally report outliers, e.g. 33.33%, 50%, 100% of ER visits being covid-related. As of May 2024, an analysis shows around 10 unusually high values across the full history of all signals, so they are rare. We expect that these high rates are by chance, due to a small total number of ED visits in a given week.
+
+
+## Missingness
+
+As of May 2024, NSSP received data from 78% of US EDs.
+The most noticeable gaps in [county coverage](https://www.cdc.gov/nssp/media/images/2024/04/Participation-with-date.png) are in California, Colorado, Missouri, Oklahoma, and Virginia, though most states have some counties that are absent.
+
+The following states have no country-level data at all: CA, WA, AK, AZ, AL, CO, SD, ND, MO, AR, FL, OH, NH, CT, NJ.
+Counties which do not have data are listed, but with an `NA` value.
+
+At the state level, South Dakota, Missouri, and territories are not reported.
+
+
+## Lag and Backfill
+
+The weekly signal is reported on Friday mornings, adding data from the prior week.
+For example, on Friday, 2024-04-19, the source added new data from the week ending 2024-04-13.
+
+This data source has frequent backfill, primarily arising from newly included EDs. When a new facility joins the reporting network, its historical data is added to the dataset, resulting in changes to historical values for every geographic level that ED is part of (county through nation). Because of this, the broadest geographic levels are more likely to be revised.
+
+In previous revisions, we have noted changes to values dating back about 2 years.
```

Comment:
Do we have a similar revision analysis posted on the other sources? I don't ever remember seeing them. I agree it would be useful, but that sounds more like an overall doc overhaul thing

Reply:
We have this type of info [for some sources](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/hospital-admissions.html#lag-and-backfill). It's something that Roni has requested more detail about recently and that we've wanted info about for various signals at various points in time.

I don't think this is critical to add now, but it will be easiest, since we've done the statistical analysis recently. I was hoping that @minhkhul or you had come across info about this during the analysis.

Questions about are some things that came to mind but are more guiding questions for the type of detail I'd be looking for. They don't all need to be answered exactly, but any additional info would be helpful.

Ultimately, this is up to y'all.